config/connect-log4j.properties

```diff
@@ -13,19 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=INFO, stdout
+log4j.rootLogger=INFO, stdout, connectAppender
 
 
```

Comment:
At the same time, it might also be useful to add a similar comment here, such as:
```
# Send the logs to the console.
#
```